frontend/src/components/TaskHeatmap.tsx

```diff
@@ -78,13 +79,15 @@ export const TaskHeatmap = () => {
             </div>
           ))}
         </div>
-        <p className={classes(css.yaxismax)}>5</p>
+        <p className={classes(css.yaxismax)}>25</p>
```

Comment:
I'd probably use `{convertScale(5)}` for these numbers, just so it's easier to remember to change if needed

Reply:
Not sure if it would be better to also "convert" the ones for consistency, even though those stay the same